src/components/application_manager/src/hmi_capabilities_impl.cc

```diff
@@ -126,6 +126,13 @@ HMICapabilitiesImpl::HMICapabilitiesImpl(ApplicationManager& app_mngr)
     is_ivi_cooperating_ = true;
     is_rc_cooperating_ = true;
   }
+  requests_required_for_capabilities_ = {
+      hmi_apis::FunctionID::VehicleInfo_IsReady,
+      hmi_apis::FunctionID::VR_IsReady,
+      hmi_apis::FunctionID::TTS_IsReady,
+      hmi_apis::FunctionID::UI_IsReady,
+      hmi_apis::FunctionID::RC_IsReady,
+      hmi_apis::FunctionID::Navigation_IsReady};
```

Comment:
@LitvinenkoIra I see that all these requests are matching bool flags like `is_rc_cooperating_`
Is it possible to remove the bool flags and track corresponding events by checking whether it is in `requests_required_for_capabilities_` or not?

Reply:
@AKalinich-Luxoft These bool flags indicate if the interface is available or not. By tracking events with requests_required_for_capabilities_, we can check if a response was received or not, but we cannot check if it was successful (I mean the interface is available) or not (the interface is not available)